scenes/menu/main_menu.gd

```diff
@@ -151,6 +151,8 @@ func update_char(label, portrait, char_id):
 	var display_name = "Random"
 	if char_id == "random_s7":
 		char_id = "random"
+	elif char_id == "random_s6":
+		char_id = "blazbluelogo2"
```

Comment:
it looked like you added the s6 logo but this is still blazblue?

Reply:
also whoops